.docs/user-guide/schedulers/docker/plug-ins/csi-nfs.md

```diff
@@ -27,3 +27,54 @@ The following environment variables can be used to configure the plug-in:
 | Environment Variable | Description | Default | Required |
 |---------------------|-------------|---------|---------|
 | `X_CSI_NFS_VOLUMES` | A list of NFS volume mappings | | |
+
+## Examples
+
+To use the `csi-nfs` plug-in, you must first create a volume using the
+`docker volume create` command (unless predefined using `X_CSI_NFS_VOLUMES`, see
+`Limitations` below), specifying both the NFS server host and the
+exported path:
+
+```sh
+$ docker volume create -d rexray/csi-nfs -o host=192.168.75.2 -o export=/data test
+$ docker volume ls
+DRIVER                           VOLUME NAME
+rexray/csi-nfs                   test
+```
+
+With the volume created, you can then use it in your containers:
+
+```sh
+$ docker run -it -v test:/mnt alpine sh
+/ # mount | grep nfs
+192.168.75.2:/data on /mnt type nfs4 (rw,relatime,vers=4.1,rsize=65536,wsize=65536,namlen=255,hard,proto=tcp,port=0,timeo=600,retrans=2,sec=sys,clientaddr=10.0.2.15,local_lock=none,addr=192.168.75.2)
+```
+
+### Limitations
+
+`csi-nfs` is an early proof of concept to demonstrate a Docker managed plug-in functioning with [CSI](https://github.com/container-storage-interface/spec). It is
+not intended to replace Docker's
+[native NFS](https://docs.docker.com/engine/reference/commandline/volume_create/#driver-specific-options)
+functionality.
+
+Currently, the plug-in only supports NFSv4, and does not allow for custom options
```

Comment:
I think one of the unique opportunities that CSI-NFS has is with non-Swarm orchestration environments. Maintaining what could amount to hundreds of volumes with `docker volume create` commands would be onerous, less so in a RR master config (if I understand the configuration correctly that there can be a central RR that others proxy to). Or am I misapplying RR in that it's really only intended as a multiplexer to reduce driver development tasks?